src/pages/steps/StepsForm.tsx

```diff
@@ -127,6 +127,14 @@ const StepsForm = ({
     },
   });
 
+  const triggerHotjarSurvey = () => {
+    if (typeof window === 'undefined') return;
+    if (offer?.mediaObject) return;
+
+    // @ts-expect-error
+    window.hj('trigger', 'no_image');
```

Comment:
Might be good to move to the [official SDK](https://hotjar.github.io/hotjar-js/) later on if it has better completion and stuff

Reply:
Oh yes good suggestion 👍